UPDATE.md

```diff
@@ -0,0 +1,56 @@
+# Update cloud-provider-gcp with Kubernetes release
+
+## Overview
+
+k8s.io/cloud-provider-gcp provides a sample distribution of how to run Kubernetes on the Google Cloud.
+This project allows a Kubernetes cluster to provision, monitor and add/remove GCP resources necessary for operation of the cluster. 
+It has to be updated together with Kubernetes releases. 
+This document describes the steps to update cloud-provider-gcp with Kubernetes release.
```

Comment:
This document describes the steps to update cloud-provider-gcp with *a* Kubernetes release ?version.

Reply:
Updated